include/tfblib/tfb_errors.h

```diff
@@ -64,6 +64,9 @@
 /// Unable to find a font matching the criteria
 #define TFB_ERR_FONT_NOT_FOUND           15
 
+/// Unable to flush the framebuffer with ioctl()
+#define TFB_ERR_FB_FLUSH_IOCTL_FAILED 16
```

Comment:
That's great, but you also need to add the string version of the error at the end of the array in `errors.c`: https://github.com/vvaltchev/tfblib/blob/master/src/errors.c

Otherwise, tfb_strerror() won't work.